src/Autocomplete/Domain/Model/Location.php

```diff
@@ -12,11 +12,8 @@ class Location {
 
 	/**
 	 * ID automagically set by Doctrine ORM if needed
-	 *
-	 * @var int|null
-	 * @phpstan-ignore-next-line
 	 */
-	private ?int $id;
+	private int $id;
```

Comment:
Please remove the comment completely. The nullable setup was chosen because `Location` is a Doctrinr entity, which *usually* comes with auto-generated IDs which need nullable IDs. In this specific case, we don't autogenerate the IDs, so making the property non-nullable is good. We might need to set it to 0 in the constructor to make PHPStorm/PHPStan happy, because the tools don't know that is a Doctrine entity whose properties are set via reflection by Doctrine and might complain about missing initialization through setter or constructor parameter

Reply:
Fixed it. Now `$id` is a part of the constructor.